Update README.md (#206)
graalvm/graalvm-demos: native-oci-container-instances/README.md

> 3. On a browser, visit the OCI dashboard and open the side menu to locate "Developer Services" -> "Containers & Artifacts" -> "Container Registry"

Reviewer: ```suggestion
3. In the Oracle Cloud Console, open the navigation menu, click **Developer Services**. Under **Containers & Artifacts**, click **Container Registry**.
```

> 4. Select the directory in which you stored your image (the location corresponds to the ```<tenancy-namespace>``` that you tagged the image with)

Reviewer: ```suggestion
4. Select the directory in which you stored your image (the location corresponds to the ```<tenancy-namespace>``` with which you tagged the image).
```

> 5. Ensure that the Access type is "Public"; if it is not, click "Actions" in the top-right corner and select "Change to public"

Reviewer: ```suggestion
5. Ensure that the Access type is "Public"; if it is not, click **Actions** in the top-right corner and select "Change to public"
```

<img width="888" alt="public repo" src="https://github.com/graalvm/graalvm-demos/assets/134104678/c57e9470-e455-401d-857d-40eb8c042591">

Create an OCI Container Instance
-------------------------

> 1. From the main OCI dashboard, open the side menu and click on "Developer Services" -> "Containers & Artifacts" -> "Container Instances"

Reviewer: ```suggestion
1. From the main OCI dashboard, open the side menu and click **Developer Services** -> **Containers & Artifacts** -> **Container Instances**
```

<img width="450" alt="Screen Shot 2023-06-06 at 4 28 20 PM" src="https://github.com/egadbois/graalvm-demos/assets/134104678/5aea839b-ab79-4eec-a0b5-74927ffde179">

> 2. Click "Create container instance"

Reviewer: ```suggestion
2. Click **Create container instance**
```

> 3. Input a name and compartment location for the instance

Reviewer: ```suggestion
3. Enter a name and compartment location for the instance
```

4. In the "Shape" section, adjust the sliders to your desired amounts of OCPUs and Memory (1 CPU and 1 GB of memory will be enough thanks to the power of native image!)
![Screen Shot 2023-07-12 at 12 45 27 PM](https://github.com/egadbois/graalvm-demos/assets/134104678/393de70f-3ed4-4b62-9816-639566edb874)

5. In the "Networking" section, you may either use a pre-existing VCN or create a new one

> 6. Leave the remaining options as their default and click "Next"

Reviewer: ```suggestion
6. Leave the remaining options as their defaults and click **Next**
```

> 7. Choose a name for the container (or leave as default) and in the "Image" section click "Select image"

Reviewer: ```suggestion
7. Choose a name for the container (or leave as default) and in the "Image" section click **Select image**
```

<img width="987" alt="Create container" src="https://github.com/graalvm/graalvm-demos/assets/134104678/092d4a12-426d-4617-b8a7-3e069d9c1550">

> 8. Locate the image that you pushed in the previous section according to the ```<tenancy-namespace>``` directory that you chose

Reviewer: ```suggestion
8. Locate the image that you pushed in the previous section according to the ```<tenancy-namespace>``` directory that you chose.
```

9. Create the container and allow it a moment to intialize
<img width="1282" alt="Create container" src="https://github.com/graalvm/graalvm-demos/assets/134104678/dba8c624-1fff-4c2c-883e-0287b910ff95">

> 10. A successful deployment will result in the large icon box turning green and the status "ACTIVE" displayed beneath it

Reviewer: ```suggestion
10. A successful deployment will result in the large icon box becoming green and the status "ACTIVE" displayed beneath it.
```

> 2. Open the side menu and click on "Developer Services" -> "Containers & Artifacts" -> "Container Registry"

Reviewer: ```suggestion
2. In the Oracle Cloud Console, open the navigation menu, click **Developer Services**. Under **Containers & Artifacts**, click **Container Registry**.
```

> 3. Select the container image that you uploaded and under the "Actions" drop down select "Delete repository"

Reviewer: ```suggestion
3. Select the container image that you uploaded and from the **Actions** drop-down list select **Delete repository**.
```

![Delete registry](https://github.com/graalvm/graalvm-demos/assets/134104678/b8135869-fd64-48f9-989a-83c188e8b5eb)

> 4. On the left side of the page, select "Container Instances"

Reviewer: ```suggestion
4. On the left side of the page, click **Container Instances**.
```

> 5. Select the container instance that you created and under the "More actions" drop down select "Delete"

Reviewer: ```suggestion
5. Select the container instance that you created and from the **More actions** drop-down list select **Delete **.
```

![Delete instance](https://github.com/graalvm/graalvm-demos/assets/134104678/cb439c80-6155-47de-956f-031469741e1f)

> 6. Open the side menu and click on "Networking" -> "Virtual cloud networks"

Reviewer: ```suggestion
6. In the Oracle Cloud Console, open the navigation menu, click **Developer Services**and then click **Virtual cloud networks**.
```

7. Select the VCN that you created for your container instance

> 8. Click the red "Delete" button

Reviewer: ```suggestion
8. Click **Delete**.
```

![Delete VCN](https://github.com/graalvm/graalvm-demos/assets/134104678/34ed6396-73f1-4f2d-9ec7-409b463b295a)

> 9. Click "Scan" - this will return a list of any lingering resources that are referencing the VCN

Reviewer: ```suggestion
9. Click **Scan** - this returns a list of any lingering resources that reference the VCN.
```

> 10. Click "Delete all" to remove the listed resources (if this fails you can select each resource individually to delete them on their own pages)

Reviewer: ```suggestion
10. Click **Delete all** to remove the listed resources (if this fails you can select each resource individually to delete them on their own pages).
```
